kong/target_service_test.go

```diff
@@ -93,20 +94,37 @@ func TestTargetsUpdate(T *testing.T) {
 	assert.NotNil(createdTarget)
 	assert.Equal(targetID, *createdTarget.ID)
 
+	// determine the target with predefined ID
```

Comment:
```suggestion
	// delete the target with predefined ID
```

?

I'm not so found of using comments in tests as they do not surface during tests runs. If we do need something to be added here I'd much rather prefer to see a log which would actually be of value when going through tests output.

Reply:
Sorry, I added the comments mostly as a personal reminder when doing the test to "split" it into section. I removed the comment as the code is relatively self-explanatory. 

Done in commit 92b49b0352904798529f112625db85a44425393e